tests/run-make/dirty-incr-due-to-hard-link/rmake.rs

```diff
@@ -0,0 +1,23 @@
+use run_make_support::{run, rustc};
```

Comment:
I could *not* make this into a `tests/incremental` test b/c it simply wouldn't reproduce, but `tests/run-make` works fine. Probably has to do with how `-Csave-temps` interacts w/ incremental tests, given that it generates temp files into the cwd.

Is there a way to limit this just to `x86_64-unknown-linux-gnu`? I think we just need one platform with asm support for this test.

Reply:
Yes, you can do `//@ only-x86_64-unknown-linux-gnu` I believe.

Reply:
Oh ok, I didn't know rmake files could take directives.

Reply:
`rmake.rs` files take a subset of directives. For instance, revisions are not supported, but the usual `{ignore,only}-*` ones do work. I want to eventually make which directives can be used in which test suites way easier to figure out, but yeah.

Reply:
gamer